torch/_meta_registrations.py

```diff
@@ -2539,7 +2540,9 @@ def meta_qconv2d_pointwise(
         )
         assert output_dtype in [torch.float32, torch.bfloat16, torch.uint8, torch.int8]
         out = x.new_empty(shape_out, dtype=output_dtype)
-        out = out.to(memory_format=torch.channels_last)
+        format = torch.channels_last if len(shape_out) == 4 else torch.contiguous_format
```

Comment:
What about Conv3D? should it be `torch.channels_last3d`? If we don't support the fusion, maybe we can add a assert to reminder when further enable it.

Reply:
Yeah. Sure. I am wondering why there was no such checks when only 2d was supported

Reply:
Check added.